tests/test_requirements.py

```diff
@@ -198,6 +203,17 @@ def test_empty_specifier(self) -> None:
         assert req.name == "name"
         assert req.specifier == ""
 
+    def test_canonical_name(self) -> None:
+        # GIVEN
+        to_parse = "split_name"
+
+        # WHEN
+        req = Requirement(to_parse)
+
+        # THEN
+        assert req.name == "split_name"
+        assert canonicalize_name(req.name) == "split-name"
```

Comment:
Maybe this test doesn't make sense now that I've removed the `.canonical_name` property?

Reply:
Removed